modules/apps/calendar/calendar-web-test/src/testFunctional/tests/NewEvent.testcase

```diff
@@ -1118,12 +1118,27 @@ definition {
 	}
 
 	@description = "LRQA-71887 - Verify that the user can select Start / End dates from the datepicker component"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanSelectStartEndFromDatepicker {
+		CalendarNavigator.gotoAddEvent();
+
+		CalendarEvent.editTitle(eventTitle = "Calendar Event Title");
+
+		CalendarEvent.setSpecificDate(
+			endDay = "14",
+			startDay = "14");
+
+		CalendarEvent.viewStartsDate(startDate = "12/14/2021");
```

Comment:
Hi @alanafergs ! If you use this static dates, the test will fail when running in different days. I changed this part of the code. See bellow:
```
                var displayDateDay = "14";
		var displayDateMonth = DateUtil.getFormattedCurrentDate("MM");	
		var displayDateYear = DateUtil.getCurrentYear();
		var displayDate = "${displayDateMonth}/${displayDateDay}/${displayDateYear}";

		CalendarEvent.viewStartsDate(startDate = "${displayDate}");

		CalendarEvent.viewEndsDate(endDate = "${displayDate}");
```